corehq/apps/hqadmin/templates/hqadmin/branches_on_staging.html

```diff
@@ -0,0 +1,10 @@
+<h1>Branches Currently Deployed</h1>
+<p>This page only means something on staging.</p>
+<ul>
+{% for branch in branches %}
+    <li>
+        <a href="https://github.com/dimagi/commcare-hq/compare/{{ branch.branch }}" target="_blank">{{ branch.branch }}</a>
+        (<a href="https://github.com/dimagi/commcare-hq/commit/{{ branch.commit }}">{{ branch.commit }}</a>)
+    </li>
+{% endfor %}
+</ul>
```

Comment:
This is a standalone html page, right?  Not added as an `{% include %}` or anything?  Could throw in some `<html><head></head><body></body></html>` tags to make that more obvious.